services/api-server/src/simcore_service_api_server/modules/director.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+import httpx
+from fastapi import FastAPI
+
+from ..core.settings import DirectorSettings
+from ..utils.client_decorators import JsonDataType, handle_errors, handle_retry
+from ..utils.client_base import BaseServiceClientApi
+
+logger = logging.getLogger(__name__)
+
+# Module's setup logic ---------------------------------------------
+
+
+def setup(app: FastAPI, settings: DirectorSettings) -> None:
```

Comment:
this is actually to talk with director-v2 no? please correct the name --> confusion